api/routes/user/login.js

```diff
@@ -11,6 +14,41 @@ router.post('/', (req, res) => {
     })
 })
 
+router.post('/', async (req,res)=>{ 
+    const{ email, password} = req.body;
+    console.log(req.body);
+
+    const user= await User.findOne({email:email});
+    console.log('user',user);
+
+    let existingUser;
+    try{
+        existingUser=await User.findOne({email:email});
+    }catch(err){
+        return res.status(400);
+    }
+
+    if(!existingUser){
+       res.status(400).send('Invalid credentials,could not log you in');
+       
```

Comment:
return statement? the rest of the code will still execute event if you sent the response. In order to stop that, you need a return statement